assets/dev/js/editor/elements/views/base.js

```diff
@@ -659,6 +673,35 @@ BaseElementView = BaseContainer.extend( {
 		this.renderHTML();
 	},
 
+	isAtomicDynamic( settingKey ) {
+		const widgetConfig = elementor.widgetsCache[ this.model.config.name ]
+		return settingKey.includes( '__dynamic__' ) &&
+			elementorCommon.config.experimentalFeatures.e_nested_atomic_repeaters &&
+			widgetConfig.supports_atomic_dynamic_titles; // May be redundant
```

Comment:
This condition may be redundant.
We need to provide one solution for titles and links as well.
I have created a separate task for the MM dynamic links.